### ISSUE TITLE
Big Sur compatibility

### ISSUE THREAD

**Issue by angietr** [graphsketcher/GraphSketcher#27]
It works in general, except that double clicking text renders it illegible with a shading of blue covering the whole text (which can still be modified using the inspect window). The real problem I faced was export. It can only export to PDF and not an image file. Checking of the signature suggests that there is possibly a permission problem. I tried resigning, permission correction, none of which helped.

**Comment by BBBernarddd** [comment]
export fails on MacBook M1. Change color of line is also no longer working.

**Comment by angietr** [comment]
> export fails on MacBook M1. Change color of line is also no longer working.

For line color: try websafe colors tab

**Comment by dustin** [comment]
This doesn't seem to be very close to building on my M1 Mac in Xcode 13 on OS X 11.5.2.  I hacked for a bit and kept getting further out of things I had confidence might work.

Does anyone have a working recipe?

**Comment by robinstewart** [comment]
Yes, there is a lot of work needed to bring the code up to date before it has any chance of building in Xcode 13. It might be possible to build it from source using an old Mac with Xcode tools from circa 2014?

Of course, you can still download the pre-built version: https://github.com/graphsketcher/GraphSketcher/releases

**Comment by PhilMac** [comment]
> For line color: try websafe colors tab

Works perfectly!

**Comment by RobertTalbert** [comment]
Export appears to be broken though; it will export to PDF but no other file format.
